doc/getting_started.md

```diff
@@ -0,0 +1,14 @@
+Getting Started
+===============
+
+This package implements a RobotDriver and several applications for the TriFinger
+robots, using the [`robot_interfaces`
+package](https://open-dynamic-robot-initiative.github.io/code_documentation/robot_interfaces/docs/doxygen/html/index.html).
+See there for more information on the general architecture.  The action and
+observation types for the (Tri)Finger robots are also implemented in
+`robot_interfaces`.
+
+For a basic example on how to control the robot, see
+[demo_real_finger.py](https://github.com/open-dynamic-robot-initiative/robot_fingers/blob/master/demos/demo_real_finger.py)
+and the other
+[demos](https://github.com/open-dynamic-robot-initiative/robot_fingers/blob/master/demos).
```

Comment:
do we also have a demo in cpp? if yes we should link it here, otherwise we should make a remark here that this can also be done in cpp, otherwise people might think they have to use python

Reply:
No, we don't have a C++ demo.  I added a comment.